### ISSUE TITLE
Issue while starting service

### ISSUE THREAD

**Issue by jamsheer-thottathil** [strapdata/elassandra#171]
We are getting following error while starting the service

```
java.lang.AbstractMethodError: org.apache.cassandra.utils.JMXServerUtils$Exporter.exportObject(Ljava/rmi/Remote;ILjava/rmi/server/RMIClientSocketFactory;Ljava/rmi/server/RMIServerSocketFactory;Lsun/misc/ObjectInputFilter;)Ljava/rmi/Remote;
        at javax.management.remote.rmi.RMIJRMPServerImpl.export(RMIJRMPServerImpl.java:150)
        at javax.management.remote.rmi.RMIJRMPServerImpl.export(RMIJRMPServerImpl.java:135)
        at javax.management.remote.rmi.RMIConnectorServer.start(RMIConnectorServer.java:405)
        at org.apache.cassandra.utils.JMXServerUtils.createJMXServer(JMXServerUtils.java:104)
        at org.apache.cassandra.service.CassandraDaemon.maybeInitJmx(CassandraDaemon.java:143)
        at org.apache.cassandra.service.CassandraDaemon.setup(CassandraDaemon.java:188)
        at org.apache.cassandra.service.ElassandraDaemon.activate(ElassandraDaemon.java:219)
        at org.apache.cassandra.service.ElassandraDaemon.main(ElassandraDaemon.java:547)

```

It seem like issue was regarding the issue available in cassandra JIRA  https://issues.apache.org/jira/browse/CASSANDRA-14173

**Comment by jamsheer-thottathil** [comment]
I could able to work it by degrading JDK  :)

**Comment by vroyer** [comment]
Fixed with release 5.5.0.13+ and upgrade to cassandra-3.11.2
